plugin-server/src/main/ingestion-queues/batch-processing/each-batch.ts

```diff
@@ -1,13 +1,13 @@
 import { EachBatchPayload, KafkaMessage } from 'kafkajs'
 
 import { status } from '../../../utils/status'
-import { groupIntoBatches } from '../../../utils/utils'
```

Comment:
That util is now completely unused, so maybe it could be just removed to avoid confusion?

Reply:
it's used for the async side